ml_datasets/loaders/mnist.py

```diff
@@ -28,10 +31,34 @@ def mnist(variant='mnist', shuffle=True):
         (X_train, y_train), (X_test, y_test) = load_fashion_mnist()
     elif variant == 'kuzushiji':
         (X_train, y_train), (X_test, y_test) = load_kuzushiji_mnist()
+    elif variant.startswith("emnist-"):
+        split = variant.split("-")[1]
```

Comment:
I don't think it's a big deal, but note that this code will work for `variant = emnist-digits-something`